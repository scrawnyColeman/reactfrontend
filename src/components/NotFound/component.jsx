import React from 'react';
import styled from 'styled-components';

const StyledNotFound = styled.div`
    font-size: 2rem;
    margin: 30vh;
`;

const NotFoundPage = () => <StyledNotFound>Oops... something went wrong! Path not found.</StyledNotFound>;

export default NotFoundPage;
