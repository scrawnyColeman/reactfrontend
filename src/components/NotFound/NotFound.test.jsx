import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import 'jest-styled-components';
import React from 'react';
import NotFound from './component';

Enzyme.configure({ adapter: new Adapter() });

describe('NotFound component', () => {
    test('component matches snapshot', () => {
        const wrapper = shallow(<NotFound />);
        expect(wrapper).toMatchSnapshot();
    });
});
