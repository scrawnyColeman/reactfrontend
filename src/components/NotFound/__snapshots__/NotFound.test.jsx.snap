// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component component matches snapshot 1`] = `
<styled.div>
  Oops... something went wrong! Path not found
</styled.div>
`;
